examples/textual_inversion/README.md

```diff
@@ -69,6 +69,7 @@ accelerate launch textual_inversion.py \
 ```
 
 A full training run takes ~1 hour on one V100 GPU.
+Add flag `--amp_data_type=bf16` if you want to use bf16 for unet forward to accelerate finetune. A full training by this flag run takes ~30 min on one V100 GPU.
```

Comment:
V100 doesn't have BF16 support, BF16 support in GPU is since A100.

Reply:
I think it works on any Ampere cards? (Including 3090, A10, A5000, A6000)